improver/wxcode/wxcode_decision_tree.py

```diff
@@ -37,7 +37,7 @@
 def wxcode_decision_tree():
     """
     Define queries that comprise the weather symbol decision tree.
-    Each queries contains the following elements:
+    Each query contains the following elements:
```

Comment:
From the perspective of grouping related nodes, it would be useful if the `drizzle_mist`, `drizzle_is_rain`, `drizzle_cloud`, `no_precipitation_cloud`, `overcast_cloud` and `partly_cloud` nodes were grouped with the `mist_conditions` and `fog_conditions` nodes at the bottom of this file. Just to note that I think that this would cause potential conflicts with #1376, which also modifies this file. What do you think?

Reply:
Yes. I thought this yesterday when updating the documentation. Since you've mentioned it too, I've sorted the tree to be success-fail order (except one mist node where it made sense to make one exception). Bit worried this might cause conflicts if anyone is working on the weather code trees though.

Reply:
As discussed in the stand-up, the reordering has been removed from this PR and will be handled in #1378.